### ISSUE TITLE
[QUESTION] Declare dependecy in cbv without assigning to value

### ISSUE THREAD

**Issue by jstoebel** [dmontagu/fastapi-utils#97]
**Description**
Is there a way to declare a dependency in a cbv without assigning a value to a class member? 

I can add dependencies in a path operation decorator like this:

```python
@app.get("/items/", dependencies=[Depends(verify_token), Depends(verify_key)])
async def read_items():
    return [{"item": "Foo"}, {"item": "Bar"}]
```

Is there a analogous way to do this with a cbv? The only way I'm aware of is to do something like this

```python
@cbv
class MyView:
    _noop = Depends(authenticate_token) # I don't need to access the return value of `authenticate_token`
```

**Additional context**
N/A

**Comment by falkben** [comment]
you want to apply it once to the entire class I assume not to each path operator? Because setting the dependencies like that at the path operator should still work w/in cbv's.

**Comment by jstoebel** [comment]
@falkben  that's correct, I'm curious if there's a way to do it once and have it apply to the entire class.

**Comment by falkben** [comment]
You could write your dependency in the __init__ method as long as you aren't specifying user input with the Depends. Or you could write it as a class attribute but just assign the output to `_`
